pkg/scheduler/scheduler.go

```diff
@@ -383,16 +383,21 @@ func (s *Scheduler) doScheduleBinding(namespace, name string) (err error) {
 		metrics.BindingSchedule(string(ScaleSchedule), utilmetrics.DurationInSeconds(start), err)
 		return err
 	}
-	// TODO(dddddai): reschedule bindings on cluster change
```

Comment:
What's your idea here? @dddddai 
We need a way to re-schedule on `cluster change`.